.ci_support/osx64.yaml

```diff
@@ -6,3 +6,5 @@ fortran_compiler:
   - gfortran
 channel_sources:
   - conda-forge,defaults
+MACOSX_DEPLOYMENT_TARGET:
+  - "10.10"
```

Comment:
Must be removed before merging to master, but is required in order for the unit-tests to work on the Mac.

Reply:
This should be 10.11 according to the docs about the binaries.

Reply:
I'm packaging 1.12.17, my understanding is that it is tied to 10.10 in that case.